apps/web/src/common/constants.ts

```diff
@@ -50,6 +52,21 @@ export const sidebarNavItems: SidebarButtonProps[] = [
     hidden: true,
     roles: [Role.USER],
   },
+  {
+    id: '004',
+    kind: SidebarButtonKind.LINE_BREAK,
+    roles: [Role.ADMIN],
```

Comment:
Nit - Does this imply that for tabs 1-3, they are accessible for only USERs and not ADMINs?

Reply:
Yes. ADMINs alone can only access this line break and user management; while USERs alone can access tab 1-3. Both roles together gives complete user access.